cockroachdb/docs
# DOC-5978: Remove unnecessary redirects.

Fixes DOC-5978

Steps to test this:

1. Check to make sure `/dev` URLs all work
2. Check to make sure `/stable` URLs all work
3. Check to make sure all redirects from `collatedstring.html` go to `collate.html` (one example)
4. Make sure htmltest passes (this is implied if the Netlify build completes)
